recursion/combinationForm/Solution1Recursion.go

```diff
@@ -0,0 +1,16 @@
+package CombinationFormula
+
+type ImplS1R struct{}
+
+func fact(n int) int {
```

Comment:
I think fact func should be part of the type as otherwise when you write another implementation for practice (in the same package) then it would interface with that. So please move into the same package.

Reply:
fixed